### PR TITLE
[WIP][12.0][MIG] shopinvader_custom_attribute

### PR DESCRIPTION
**Pull request by hparfr**
In some searchengine like elastic, you can't have in an array different types for the same value.
Therefore, in this fix, we always stringify the values and declare the type.

### CODE REVIEW

File: shopinvader_custom_attribute/__manifest__.py [hunk 1]
```diff
@@ -6,13 +6,13 @@
 {
     "name": "Shopinvader Custom Attribute",
     "summary": "Integrate your custom attribute in your website",
-    "version": "10.0.1.0.0",
```
Comment (simahawk) [inline shopinvader_custom_attribute/__manifest__.py:9]: which makes this a migration + fix I guess? :smile:

File: shopinvader_custom_attribute/__manifest__.py [hunk 2]
```diff
@@ -6,13 +6,13 @@
...
     "category": "e-commerce",
     "website": "https://akretion.com",
     "author": "Akretion",
     "license": "AGPL-3",
-    "application": True,
-    "installable": False,
+    "application": False,
+    "installable": True,
     "external_dependencies": {"python": [], "bin": []},
     "depends": ["shopinvader", "product_custom_attribute"],
```
Comment (simahawk) [inline shopinvader_custom_attribute/__manifest__.py:17]: this module comes from you own repo and is not migrated yet https://github.com/shopinvader/odoo-pim

**Review by simahawk — CHANGES_REQUESTED** [review]

File: shopinvader_custom_attribute/__manifest__.py (hunk 2 above)
Comment (hparfr) [reply shopinvader_custom_attribute/__manifest__.py:17]: Thanks. I was starting from an early migr in odoo-ak-incubator.
In odoo-pim the pr is close to be merged https://github.com/shopinvader/odoo-pim/pull/6

Indeed the current PR is waiting for his dependency.

**Comment by sebastienbeau** [comment]
@hparfr dependency have been merged can you rebase it?

**Comment by hparfr** [comment]
Closing due to inactivity. custom_attribute as been moved to pim. It's better to start a fresh pr.

**Pull request closed without merging**